policy/modules/system/systemd.te

```diff
@@ -1417,3 +1417,516 @@ optional_policy(`
+allow systemd_sleep_t systemd_homed_t:dbus send_msg;
+
+#### Allow: Reboot/login
+# AVC avc:  denied  { send_msg } for  scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=dbus permissive=0
+# AVC avc:  denied  { connectto } for  pid=1000 comm="sshd" path="/run/systemd/userdb/io.systemd.Home" scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=unix_stream_socket permissive=0
+optional_policy(`
+	gen_require(`
+		type sshd_t;
+	')
+	allow sshd_t systemd_homed_t:dbus send_msg;
+	systemd_homed_userdb_stream_connect(sshd_t)
+')
+
+# AVC avc:  denied  { send_msg } for  scontext=system_u:system_r:xdm_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=dbus permissive=0 exe="/usr/bin/dbus-broker" sauid=81 hostname=? addr=? terminal=?
+# AVC avc:  denied  { connectto } for  pid=1000 comm="gdm-session-wor" path="/run/systemd/userdb/io.systemd.Home" scontext=system_u:system_r:xdm_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=unix_stream_socket permissive=0
+# AVC avc:  denied  { write } for  pid=1000 comm="gdm-session-wor" path="/run/systemd/home/testuser.dont-suspend" dev="tmpfs" ino=3614 scontext=system_u:system_r:xdm_t:s0-s0:c0.c1023 tcontext=system_u:object_r:systemd_homed_runtime_t:s0 tclass=fifo_file permissive=0
+optional_policy(`
+	gen_require(`
+		type xdm_t;
+	')
+	xserver_dbus_chat_xdm(systemd_homed_t)
+	systemd_homed_userdb_stream_connect(xdm_t)
+	allow xdm_t systemd_homed_runtime_t:fifo_file write;
+')
+
+# AVC avc:  denied  { send_msg } for  scontext=system_u:system_r:local_login_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=dbus permissive=0
+# AVC avc:  denied  { connectto } for  pid=1000 comm="login" path="/run/systemd/userdb/io.systemd.Home" scontext=system_u:system_r:local_login_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_homed_t:s0 tclass=unix_stream_socket permissive=0
+# AVC avc:  denied  { write } for  pid=1000 comm="login" path="/run/systemd/home/testuser.dont-suspend" dev="tmpfs" ino=4784 scontext=system_u:system_r:local_login_t:s0-s0:c0.c1023 tcontext=system_u:object_r:systemd_homed_runtime_t:s0 tclass=fifo_file permissive=0
+optional_policy(`
+	gen_require(`
+		type local_login_t;
+	')
+	allow local_login_t systemd_homed_t:dbus send_msg;
+	systemd_homed_userdb_stream_connect(local_login_t)
+        allow local_login_t systemd_homed_runtime_t:fifo_file write;
+')
+
+
+#
+# /home and /run/systemd/user-home-mount
+#
+
+# Select context for /run/systemd/user-home-mount as user_home_dir_t as it
+# acts as "proto" directory where homed-directories are initially
+# created and set up. It acts as essentially the same as /home. This
+# also shortens this policy as now we can share some rules as /home.
+filetrans_pattern(systemd_homed_t, init_var_run_t, user_home_dir_t, dir, "user-home-mount")
```

Comment:
This is probably just internal homed runtime.  Probably shouldnt be labeled user_home_dir_t. But this does raise a question though. Something needs to label these filesystems. I think this should probably be done manually by either homed or homework. (I use to label them manually myself on first login)